### DISCUSSION TITLE
Different UI tags locally than in production

### DISCUSSION THREAD

**Discussion by shanebusenitz** [dagster-io/dagster#22059] [Q&A] [answered]
Upgraded to 1.6.4 / 0.22.4 a while back and I've noticed that locally when running partition assets I get a nice footer in the Target column on the Runs tab showing the partition.

![Screenshot 2024-05-23 at 9 57 30 AM](https://github.com/dagster-io/dagster/assets/115168327/057121f0-ac66-4427-adc6-295f309f60ca)

Great UI feature and saves me clicking into the tags. However in production (which is running the exact same version of dagster) I'm seeing only the combined tags on the very same asset partition run. 

![Screenshot 2024-05-23 at 10 03 42 AM](https://github.com/dagster-io/dagster/assets/115168327/ddf1afe2-8698-4980-8230-93cb17c9761e)

I'm running production in kubernetes and I can see some of the additional tags there are releated to kubernetes (ie. image name). Is there a way I can change this so that I can get the partitions to show seperate from the tags like it does locally?

**Comment by tacastillo** [comment, marked as answer]
Hey Shaneb! Thanks for reaching out. You know...now that I think about it, that's a good point.

I believe you picked up why it happens, but to clarify it for future readers:

So, when you're running Dagster in production, you have it set up to execute your runs in a Kubernetes pod. Dagster surfaces the relevant metadata about that pod as tags. However, when running locally, everything happens in-process without extra containerization.

When running locally, because of how few tags there are, you can see the partition tag upfront and work with it easily. However, when there are more tags, our UI collapses them, preventing you from seeing the tag you originally saw locally without having to click in.


I think it's fair for you to make this into a GitHub issue that our UI and design team can think about, because you're right, there is a hierarchy of priorities for tags and some tags may want to be prioritized as visible when you have many.

I'll convert this into an issue and you can fill out any other details you'd like!
